frontend/src/components/pages/Scheduling/ConfirmDetails.tsx

```diff
@@ -271,30 +289,69 @@ const ConfirmDetails = ({
           {currentSchedule.volunteerNeeded && (
             <>
               <Text textStyle="mobileSmall" color="hubbard.100" pt="1.4em">
-                Pickup required
+                Assistance type
               </Text>
               <Text textStyle="mobileBody">
-                {currentSchedule.isPickup ? "Yes" : "No"}
+                Food rescue {currentSchedule.isPickup ? "pickup" : "unloading"}
               </Text>
-            </>
-          )}
 
-          {currentSchedule.volunteerNeeded && currentSchedule.isPickup && (
-            <Box>
+              <Text textStyle="mobileSmall" color="hubbard.100" pt="1.4em">
+                Volunteer request time
+              </Text>
+              <Text textStyle="mobileBody">{dateText(startDateLocal)}</Text>
+              <Text textStyle="mobileBody">
+                {currentSchedule.volunteerTime
+                  ? format(
+                      parse(currentSchedule.volunteerTime, "kk:mm", new Date()),
+                      "h:mm aa",
+                    )
+                  : "-"}
+              </Text>
               <Text textStyle="mobileSmall" color="hubbard.100" pt="1.4em">
                 Address
               </Text>
               <Text textStyle="mobileBody">
-                {currentSchedule.pickupLocation}
+                {currentSchedule.isPickup
+                  ? `${currentSchedule.pickupLocation}`
+                  : "Community Fridge"}
+              </Text>
+              <Text textStyle="mobileSmall" color="hubbard.100" pt="1.4em">
+                Additional notes
+              </Text>
+              <Text textStyle="mobileBody">
+                {currentSchedule.notes === "" || currentSchedule.notes === null
+                  ? "-"
+                  : currentSchedule.notes}
+              </Text>
+              <Text textStyle="mobileSmall" color="hubbard.100" pt="1.4em">
+                Assigned volunteer
+              </Text>
+              <Text textStyle="mobileBody">
+                {currentSchedule.volunteerId === undefined ||
```

Comment:
can we do something like ```currentSchedule.volunteerId == null``` since `null == undefined`? this logic seems like it's being used a lot